index.html

```diff
@@ -70,5 +70,198 @@ <h2 class="first-title">I'm Peter <br/> Glad to see you! </h2>
+            <ul class="grid">
+                <li class="tag html-tag">html</li>
+                <li class="tag css-tag">css</li>
+                <li class="tag js-tag">javascript</li>
+            </ul>
+            <div class="works-button ">
+                <button type="button" class="normal-button1">See Project</button>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="about-me-section" id="about-me">
+    <div class="container-2">
+        <div class="about-head">
+            <h1 class="first-title2">About Myself</h1>
+            <div>
+                <p class="description4">Hello I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </p>
+            </div>
+            <p class="description5">LET'S CONNECT</p>
+            <ul class="linkSocials2">
+            <li>
+                <a href="twitter.com">
+                    <img src="/Images/SocialMediaIcons/Twitter.png" alt="Twitter"/>
+                </a>
+            </li>
+            <li>
+                <a href="linkedIn.com">
+                    <img src="/Images/SocialMediaIcons/Linkedin.png" alt="linkedIn"/>
+                </a>
+            </li>
+            <li>
+                <a href="Medium">
+                    <img src="/Images/SocialMediaIcons/Medium.png" alt="Medium"/>
+                </a>
+            </li>
+            <li>
+                <a href="Vector1">
+                    <img src="/Images/SocialMediaIcons/Vector1.png" alt="Vector1"/>
+                </a>
+            </li>
+            <li>
+                <a href="Vector2">
+                    <img src="/Images/SocialMediaIcons/Vector2.png" alt="Vector2"/>
+                </a>
+            </li>
+            </ul>
+            <div class="about-me-button">
+                <button type="button" class="normal-about-me"><b>Get my resume</b></button>
+            </div>
+        </div>
+        <div class="skills-list">
+            <ul class="frame1">
+                <li class="frame-list">
+                    <div class="languages">
+                        <div class="lang">Languages</div>
+                        <div class="arrow">
+                            <img src="/Images/arrowdown.png" alt="arrowdown"/>
+                        </div>
+                    </div>
+                    <ul class="frame2">
+                        <li class="language1 list">
+                            <div><img src="/Images/Javascript.png" class="javascript" alt="javascript"/></div>
+                            <div>
+                                <p class="language">Javascript</p>
+                            </div>
+                        </li>
```

Comment:
- [ ] Please center the content of the container vertically here. You can check the Figma design.